lib/Serialization/InlinableText.cpp

```diff
@@ -0,0 +1,163 @@
+#include "swift/AST/ASTContext.h"
```

Comment:
…and it should include InlinableText.h first, since that's the header associated with this file. (Which would catch some missing includes in that header!)